zerver/lib/push_notifications.py

```diff
@@ -610,7 +610,20 @@ def send_notifications_to_bouncer(
         "apple_devices": [device.token for device in apple_devices],
     }
     # Calls zilencer.views.remote_server_notify_push
-    response_data = send_json_to_push_bouncer("POST", "push/notify", post_data)
+
+    try:
+        response_data = send_json_to_push_bouncer("POST", "push/notify", post_data)
+    except PushNotificationsDisallowedByBouncerError as e:
+        logger.error("Bouncer refused to send push notification: %s", e.reason)
```

Comment:
Is `error` the right log level? I wonder if `warning`, which isn't going to email you, is more appropriate, given that the information is also displayed in the product itself.

Reply:
Hmm yeah agreed, fixed